ding/policy/dqn.py

```diff
@@ -519,11 +519,13 @@ def _get_encoding_size(self):
             "obs": torch.randn(1, *obs),
             "next_obs": torch.randn(1, *obs),
         }
+        if self._cuda:
+            test_data = to_device(test_data, self._device)
         with torch.no_grad():
-            x, y = self._aux_encode(test_data)
+            x, y = self._model_encode(test_data)
         return x.size()[1:], y.size()[1:]
 
-    def _aux_encode(self, data):
+    def _model_encode(self, data):
         x = self._model.encoder(data["obs"])
```

Comment:
add a assertion to give more information for model definition

Reply:
done